src/test/java/org/gridsuite/gateway/ElementAccessControlTest.java

```diff
@@ -552,6 +552,17 @@ public void testDeleteElements() {
             .expectStatus().isOk();
     }
 
+    @Test
+    public void testSupervisionEndpointsAccess() {
+        initStubForJwk();
+
+        // Test access to a supervision endpoint (should be forbidden)
+        webClient.get().uri("study/v1/supervision/studies")
+            .header("Authorization", "Bearer " + tokenUser1)
+            .exchange()
+            .expectStatus().isForbidden();
+    }
+
```

Comment:
Add a test that contains the work "supervision" but that must not be blocked to show it don't block too much.